molecule/rust/molecule.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This molecule configuration file is suitable for testing Ansible
+# roles that _do not_ require SystemD.  If your Ansible role _does_
+# require SystemD then you should use molecule-with-systemd.yml
+# instead.
+#
+# Note that the molecule configuration file that is symlinked to
+# molecule.yml is the one that will be used.
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  # Debian Buster does not have a new enough version of cargo to
+  # support the aardwolf dependency of CrackMapExec, even in
+  # backports.
```

Comment:
Since the converge configuration uses [RustScan](https://github.com/RustScan/RustScan) do these comments for disabled platforms still stand?

Reply:
Please see commit 7dd16e8.